view/frontend/templates/catalog/product/view/product-protection-offer.phtml

```diff
@@ -7,22 +7,24 @@
 /** @var $block \Magento\Catalog\Block\Product\View */
 
 $categoryName = '';
+/** encoding to prevent whitespace and other abnormal characters from being used in a div id */
+$encodedProductSku = urlencode($product->getSku());
```

Comment:
i think this will need to be `urlencode($block->getProduct()->getSku())`?

could also just $product = `$block->getProduct()` before this so that $product can be used here and below where `$category` is set